webapi/src/test/scala/org/knora/webapi/responders/v2/OntologyResponderV2Spec.scala

```diff
@@ -2366,6 +2366,221 @@ class OntologyResponderV2Spec extends CoreSpec() with ImplicitSender {
+        ) should ===(true)
+        val metadata = externalOntology.ontologyMetadata
+        val newAnythingLastModDate = metadata.lastModificationDate.getOrElse(
+          throw AssertionException(s"${metadata.ontologyIri} has no last modification date")
+        )
+        assert(newAnythingLastModDate.isAfter(anythingLastModDate))
+        anythingLastModDate = newAnythingLastModDate
+      }
+
+      // Check that the corresponding partOfValue was created
+      val partOfValuePropertyIri = AnythingOntologyIri.makeEntityIri("partOfValue")
+
+      val partOfValuePropertyInfoContent = partOfPropertyInfoContent.copy(
+        propertyIri = partOfValuePropertyIri,
+        predicates = Map(
+          OntologyConstants.Rdf.Type.toSmartIri -> PredicateInfoV2(
+            predicateIri = OntologyConstants.Rdf.Type.toSmartIri,
+            objects = Seq(SmartIriLiteralV2(OntologyConstants.Owl.ObjectProperty.toSmartIri))
+          ),
+          OntologyConstants.KnoraApiV2Complex.SubjectType.toSmartIri -> PredicateInfoV2(
+            predicateIri = OntologyConstants.KnoraApiV2Complex.SubjectType.toSmartIri,
+            objects = Seq(SmartIriLiteralV2(AnythingOntologyIri.makeEntityIri("partThing")))
+          ),
+          OntologyConstants.KnoraApiV2Complex.ObjectType.toSmartIri -> PredicateInfoV2(
+            predicateIri = OntologyConstants.KnoraApiV2Complex.ObjectType.toSmartIri,
+            objects = Seq(SmartIriLiteralV2(OntologyConstants.KnoraApiV2Complex.LinkValue.toSmartIri))
+          ),
+          OntologyConstants.Rdfs.Label.toSmartIri -> PredicateInfoV2(
+            predicateIri = OntologyConstants.Rdfs.Label.toSmartIri,
+            objects = Seq(
+              StringLiteralV2("is part of", Some("en")),
+              StringLiteralV2("ist Teil von", Some("de"))
+            )
+          ),
+          OntologyConstants.Rdfs.Comment.toSmartIri -> PredicateInfoV2(
+            predicateIri = OntologyConstants.Rdfs.Comment.toSmartIri,
+            objects = Seq(
+              StringLiteralV2("Represents a part of a whole relation", Some("en")),
+              StringLiteralV2("Repräsentiert eine Teil-Ganzes-Beziehung", Some("de"))
+            )
+          ),
+          OntologyConstants.SalsahGuiApiV2WithValueObjects.GuiElementProp.toSmartIri -> PredicateInfoV2(
+            predicateIri = OntologyConstants.SalsahGuiApiV2WithValueObjects.GuiElementProp.toSmartIri,
+            objects = Seq(SmartIriLiteralV2("http://api.knora.org/ontology/salsah-gui/v2#Searchbox".toSmartIri))
+          )
+        ),
+        subPropertyOf = Set(OntologyConstants.KnoraBase.IsPartOfValue.toSmartIri)
+      )
+
+      val partOfValuePropGetRequest = PropertiesGetRequestV2(
+        propertyIris = Set(partOfValuePropertyIri),
+        allLanguages = true,
+        requestingUser = anythingAdminUser
+      )
+
+      responderManager ! partOfValuePropGetRequest
+
+      expectMsgPF(timeout) { case msg: ReadOntologyV2 =>
```

Comment:
If possible I would use more descriptive name instead of `expectMsgPF`

Reply:
I don't even know what PF means, I just copy-pasted that ;) I will change it.

Reply:
Ah, I just see now that this is a method from Akka TestKit. So, I can't change it. It's what it is called.